### ISSUE TITLE
Final Project and OLS

### ISSUE THREAD

**Issue by HackerMan1993** [HackerMan1993/Sports_Revenue_Fans_Etc#3]
Hi again @AaronGullickson . I am finishing up my project as well as the (now ancient) OLS assignment. I should have both in to you by Friday, with me trying to finish the final by tomorrow night. I've been having a grading debacle in the undergrad stats class with people trying to stretch the truth of what Caleb has told them is acceptable versus what he actually told them; I think it'll be sorted out by tomorrow. Sorry again for the delays!

**Comment by HackerMan1993** [comment]
@AaronGullickson will you be on campus at all today? I've finished the OLS assignment but I can't knit and it's driving me insane.

**Comment by AaronGullickson** [comment]
@HackerMan1993 I will be, but you could also just push up what you have and I could probably diagnose the problem in five minutes. If its the OLS regression project, you should open the issue on that repository.

**Comment by HackerMan1993** [comment]
@AaronGullickson I have finished the paper. Of course, I am having problems with both knitting and the same push/pull problems that always plague me (I saved, checked the boxes, committed and commented, amended previous commits, and pushed and still managed to screw it up so I don't know what step I seem to always forget.) Let me know what I can do to get the paper to you as quick as possible. And thank you for your patience.
![Why Can I Never Merge Properly](https://user-images.githubusercontent.com/47223689/59552811-5ba91f80-8f40-11e9-8a89-1accddb580f5.png)
